glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -368,6 +380,30 @@ open class GleanInternalAPI internal constructor () {
         LibGleanFFI.INSTANCE.glean_destroy_glean(handle, e)
         handle = 0L
     }
+
+    /**
+     * Register a [PingType] in the registry associated with this [Glean] object.
+     */
+    internal fun registerPingType(pingType: PingType) {
```

Comment:
Does this need to be `@Synchronized` together with `initialize`?

Reply:
Good point.  I think it probably does.

Reply:
This needs to be `@Synchronized` as well.